fhttp/http_test.go

```diff
@@ -248,6 +248,57 @@ func TestDebugSummary(t *testing.T) {
 	}
 }
 
+func TestParseStatus1(t *testing.T) {
+	var tests = []struct {
+		input    string
+		expected int
+	}{
+		// Error cases
+		{"x", 400},
+		{"1::", 400},
+		{"x:10", 400},
+		{"555:-1", 400},
+		{"555:101", 400},
+		{"551:45,551:56", 400},
+		// Good cases
+		{"555", 555},
+		{"555:100", 555},
+		{"555:0", 200},
+		{"551:45,551:55", 551},
+	}
+	for _, tst := range tests {
+		if actual := parseStatus(tst.input); actual != tst.expected {
+			t.Errorf("Got %d, expected %d for parseStatus(%q)", actual, tst.expected, tst.input)
+		}
+	}
+}
+
+// Round down to the nearest hundred
+func roundhundred(x int) int {
+	return int(float64(x)+50.) / 100
+}
+
+func TestParseStatus2(t *testing.T) {
```

Comment:
nit: maybe call TestStatusDistribution ?  that would help immediately clarify the intent.